dataeng/jobs/analytics/ReadReplicaExportToS3.groovy

```diff
@@ -73,7 +73,10 @@ class ReadReplicaExportToS3 {
                     if (db_config.get('SNITCH')) {
                         shell('curl https://nosnch.in/' + db_config.get('SNITCH'))
                     }
-                }
+                    if (db_config.get('OPSGENIE_HEARTBEAT_NAME') && db_config.get('OPSGENIE_HEARTBEAT_KEY')){
```

Comment:
I'm not sure that the HEARTBEAT_KEY will show up in db_config -- I expect it in allVars.

Reply:
Did you resolve this because it's not an issue in this case?